test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -10,6 +10,26 @@
 
 public class UtilsTest {
 
+    @Test
+    public void tryAnyNullBehaviour() throws Exception {
+
+        // only null objects
+        assertSomeNull(null, null);
+
+        // one null object out of a mix
+        assertSomeNull("asdf", 1, null);
+        // no null objects
+        assertNoNull("1", 1, 5);
+    }
+
+    private void assertNoNull(Object... objects) {
+        assertFalse(Utils.isAnyNull(objects));
+    }
+
+    private void assertSomeNull(Object... objects) {
+        assertTrue(Utils.isAnyNull(objects));
+    }
+
```

Comment:
In general, a PR should not contain unmerged code from other PRs. Try to limit each PR to one enhancement, which means the PR should be based on a branch that starts off from the `master` branch that is in sync with the upstream `master` branch